app/components/meeting/new-meeting-modal.js

```diff
@@ -169,12 +174,14 @@ export default class MeetingNewMeetingModal extends Component {
     return agendaitem;
   }
 
+  async isAnnexMeetingKind(kind) {
```

Comment:
This async function still annoys me a bit because of the complex `await` constructions in the template that are caused by this. The only alternative I currently see is defining a `@tracked`-property `isAnnexMeeting` on this component and update it each time `this.kind` changes. Since `this.kind` only gets updated in the `setKind` action it might simplify the code.